### ISSUE TITLE
Service Module LOX boiloff

### ISSUE THREAD

**Issue by raidernick** [KSP-RO/RealismOverhaul#506]
The SM LOX boiloff is a bit too fast. You get noticeable decrease in fuel over just a few orbits. On a lunar mission the boiloff would be significant

**Comment by NathanKell** [comment]
Thanks for the poke. :)

This is in reference to Block D on N1/L3 missions, for the record. Also poking @stratochief66 who worked on the numbers as well.

**Comment by stratochief66** [comment]
I found the SM LOX to boiloff relatively slow, but a bit more than it would have in reality. I couldn't find super concrete reference values for real fuel cell liquids values on the SM, so I just added enough to make them survive 14 days with the current boiloff.

I see the current LOX boiloff values being a big problem for Block D on N1/L3 missions, as @NathanKell mentioned. Honestly, when I was testing other aspects of those missions while working on configs I would just disable boiloff.
